src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,190 @@
+
+Simple modals should have a title, a descriptive message, and action buttons for users to respond to the prompt or dismiss the dialog. These modals should be no larger than is necessary to contain their contents.
+
+Modals are also great for presenting single-screen tasks to user, such as changing a password.
+
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[DialogTask]}
+        caption={[
+            'A modal dialog can present a single-screen workflow, such as changing a password.',
+            '1. On desktop, this can be presented as a modal.',
+            '2. In mobile applications, modal dialogs should be used primarily for notifications and confirmations. Single-screen tasks or workflows are better presented as separate screens or full-screen bottom sheets.',
+        ]}
+    />
+    <MaterialDesignDescription
+        title={'Forms'}
+        url={'/patterns/forms'}
+        description={`When using form content in an overlay, be sure to follow PX Blue's guidelines for forms.`}
+        avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    />
+</React.Fragment>
+
+Modal dialogs are always blocking — they must be dismissed before the user can interact with the rest of the application content, typically by clicking a dismiss / close button or by clicking elsewhere on the screen.
+
+> If you are collecting a lot of information, it may be better to move this form to a separate page in your application rather than trying to place too much content into a modal.
 
 ## Sidebar
 
-The most common example of a sidebar is the application navigation menu. Typically, this opens on the side of the screen and is used to navigate to the different areas of the application.
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[SidePanel]}
+        caption={'Sidebars can be used to show additional information about a selected item.'}
+    />
+    <MaterialDesignDescription title={'Sheets: Side'} url={'https://material.io/components/sheets-side'} />
+</React.Fragment>
 
-Sidebars are also used extensively for displaying information about selected items in the main interface (e.g., selecting an item in a list would open a sidebar on the right with additional details about the selected item). They can also present simple form data and inputs (such as radio buttons, checkbox, dropdown menus, etc.) for editing a selected item. This type of sidebar should be temporary such that the user can easily close it with a close button or clicking elsewhere on the screen.
+Sidebars (Side Sheets in the Material documentation) are used to present supplemental information to users. Most commonly, they display information about a selected item in the main interface (e.g., selecting an item in a list would open a sidebar on the right with additional details about the selected item). However, they can also be used standalone, such as the legend for a chart or map.
```

Comment:
"standalone" feels like a strange word here, but Im not sure of a better wording